ci/docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG \
   RUST_NIGHTLY_VERSION= \
   GOLANG_VERSION=1.21.3 \
   NODE_MAJOR=18 \
-  SCCACHE_VERSION=v0.8.1 \
+  SCCACHE_VERSION=v0.9.1 \
```

Comment:
this newer version start to show hit rates, which is handy to some extent.

Reply:
bumped the version here: https://github.com/anza-xyz/agave/pull/4655